packages/changed-elements-react/src/clients/ITwinChangedElementsClient.ts

```diff
@@ -0,0 +1,86 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import type {
+  ChangedElements, ComparisonJobClient, ComparisonJob, GetComparisonJobParams, GetComparisonJobResultParams,
+  PostComparisonJobParams,
+  DeleteComparisonJobParams
+} from "./ChangedElementsClient.js";
+import { callITwinApi } from "./iTwinApi.js";
+export interface ITwinChangedElementsClientParams {
+  baseUrl: string;
+  getAccessToken: () => Promise<string>;
+}
+
+export class ITwinComparisonJobClient implements ComparisonJobClient {
+  private acceptHeader = "application/vnd.bentley.itwin-platform.v2+json";
+  private baseUrl: string;
+  private getAccessToken: () => Promise<string>;
+
+  constructor(args: ITwinChangedElementsClientParams) {
+    this.baseUrl = args.baseUrl;
+    this.getAccessToken = args.getAccessToken;
+  }
+
+  deleteComparisonJob(args: DeleteComparisonJobParams): Promise<void> {
```

Comment:
As long as it actually throws, I'm fine with that. You probably want to also add documentation on all these operations because this client could be used separately from the UI.